src/shared/identityUtils.ts

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (c) 2024, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+/* eslint-disable @typescript-eslint/no-unsafe-assignment */
+/* eslint-disable @typescript-eslint/no-unsafe-return */
+
+import { randomBytes } from 'node:crypto';
+import { DevServerUtils } from './devServerUtils.js';
+
+class LwrConfigFile {
+  public identityToken?: string;
+}
+
+// **********************************************************************************************
+// * TODO: When we finalize the implementation for the preview commands and things settle down, *
```

Comment:
IMO we shouldn't wait for things to settle down, for this functionality. The heart of this functionality is just a service that creates a CSPRNG-strong token. That core is not going to change, and should be homed in `CryptoUtils`. How we consume it and apply it to the dev server can remain in this package, and we can finalize those details when we know how we're storing the data.